src/asmcnc/apps/systemTools_app/screens/screen_build_info.py

```diff
@@ -674,6 +700,9 @@ def update_strings(self):
         self.zhead_header.text = self.l.get_str('Z head')
         self.hardware_header.text = self.l.get_str('Hardware')
 
+        enabled_text = 'enabled' if self.m.is_service_running('ssh') else 'disabled'
+        self.toggle_ssh_button.text = 'SSH: ' + enabled_text
+
```

Comment:
This needs to be localized - send new strings to Gemma & ask her to update the spreadsheet.